lib/ratchet/component.js

```diff
@@ -38,11 +42,11 @@ function Component(template, name, channel) {
     componentWillMount: function() {
       var self = this;
       channel.data(function(payload) {
-        self.setState({data: payload[name]});
+        self.setState({data: payload});
       });
     },
     render: function() {
-      return createComponent(template, this.state.data);
+      return createComp(template, this.state.data, createElement);
```

Comment:
Having trouble finding `createComp` in all these commits, so I'll leave a note here now just in case! What's the thinking behind abbreviating that method name? Do `createComponent` and `createComp` both exist, or is `createComp` the new name for `createComponent`? Either way I'm not sure I follow the thinking.